client/verta/tests/conftest.py

```diff
@@ -262,7 +262,10 @@ def client(host, port, email, dev_key):
 def experiment_run(client):
     client.set_project()
     client.set_experiment()
-    return client.set_experiment_run()
+    
+    run = client.set_experiment_run()
+    print("[TEST LOG] Run ID is {}".format(run.id))
```

Comment:
Can we spend like 30min and review that we print all the basic information we'd want? I'd prefer for us to avoid having to revisit this every time we identify we don't print something that would clearly help debugging.

Reply:
Could I take that on tomorrow?

Reply:
Of course!